database/rrdcalc.c

```diff
@@ -398,6 +400,7 @@ inline RRDCALC *rrdcalc_create_from_template(RRDHOST *host, RRDCALCTEMPLATE *rt,
     rc->hash = simple_hash(rc->name);
     rc->chart = strdupz(chart);
     rc->hash_chart = simple_hash(rc->chart);
+    uuid_copy(rc->config_hash_id, *((uuid_t *) &rt->config_hash_id));
```

Comment:
Wouldn't just `uuid_copy(rc->config_hash_id,  rt->config_hash_id)`  work?

Reply:
Yes, it does, changed, thanks.